library/src/main/java/org/xmtp/android/library/ConversationV2.kt

```diff
@@ -50,12 +51,13 @@ data class ConversationV2(
                 context = invitation.context,
                 peerAddress = peerAddress,
                 client = client,
+                createdAt = header.createdNs,
                 header = header,
             )
         }
     }
 
-    val createdAt: Date = Date(header.createdNs / 1_000_000)
+    val conversationCreatedAt: Date = Date((createdAt ?: 0L) / 1_000_000)
```

Comment:
Lets keep this named the same and fall back to the header if not defined.

```suggestion
    val createdAt: Date = Date((createdAt ?: header.createdNs) / 1_000_000)
```